packages/react-components/react-tabster/src/hooks/useKeyboardNavAttribute.ts

```diff
@@ -2,7 +2,7 @@ import { createKeyborg } from 'keyborg';
 import { useEffect, useMemo, useRef } from 'react';
 import { KEYBOARD_NAV_ATTRIBUTE } from '../focus/constants';
 import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
-import type { KeyborgCallback } from 'keyborg/dist/Keyborg';
```

Comment:
can we establish strict API contract on keyborg side so this kind of imports will explode ?

Reply:
ping ^ @ling1726  🙏

Reply:
@hotell microsoft/keyborg#21